packages/core/src/components/progress/progressBar.tsx

```diff
@@ -12,6 +12,18 @@ import { IIntentProps, IProps } from "../../common/props";
 import { clamp } from "../../common/utils";
 
 export interface IProgressBarProps extends IProps, IIntentProps {
+    /**
+     * Whether the background should animate.
+     * @default true
+     */
+    animate?: boolean;
+
+    /**
+     * Whether the background should be striped.
+     * @default true
+     */
+    stripes?: boolean;
```

Comment:
what's the new pattern? any modifier class (even non generic) should be mirrored as a prop?

Reply:
i don't understand the question. yes?

Reply:
ok. yeah it just feels super strange, gotta get used to it